crates/sui-bridge-indexer/src/eth_worker.rs

```diff
@@ -59,204 +63,355 @@ impl EthBridgeWorker {
         })
     }
 
-    pub async fn start_indexing_finalized_events(
+    pub async fn sync_events(
```

Comment:
is this function solely for backfilling old events (it's given a static range to sync) and will exit once the work is done?